learn/getting_started/algolia_migration.md

```diff
@@ -0,0 +1,249 @@
+# Migrating from Algolia to Meilisearch
+
+This guide will walk you through exporting your Algolia data and indexing it in Meilisearch using a Node.js script. To skip directly to the finished script, [click here](#finished-script).
+
+This guide also includes comparisons of Meilisearch and Algolia's [APIs](#apis), [index settings, parameters](#settings-and-parameters), and [support for the `instantsearch.js` library](#front-end-components).
+
+This is a practical guide for current Algolia users making the transition to Meilisearch. We also provide [a high-level comparison of the two search companies and their products](/learn/what_is_meilisearch/comparison_to_alternatives.md#meilisearch-vs-algolia) elsewhere.
+
+## Migration script
+
+Migrating indexes from Algolia to Meilisearch is a straightforward process:
+
+1. Export your data stored in Algolia
+2. Upload your data to Meilisearch
+3. (optional) Configure your settings to approximate the settings of your Algolia index
+
+This guide uses a [Node.js](https://nodejs.org/en/) script to upload Algolia index data to Meilisearch. If you would like to write this script in another language, we provide [SDKs for most common languages](/learn/what_is_meilisearch/sdks.md) and [welcome contributions to this guide](https://github.com/meilisearch/documentation/edit/master/learn/getting_started/algolia_migration.md).
+
+Before continuing, make sure you have both Meilisearch and Node.js installed, and access to a command-line terminal. If you're unsure, see our [quick start](/learn/getting_started/quick_start.md) for instructions on installing Meilisearch.
+
+### Initialize project
+
+Start by creating a directory `algolia-meilisearch-migration` and initializing it as an npm package:
+
+```bash
+mkdir algolia-meilisearch-migration
+cd algolia-meilisearch-migration
+npm init -y
+```
+
+This will set up the environment we need to install dependencies.
+
+Next, create a `script.js` file:
+
+```bash
+touch script.js
+```
+
+This file will contain our migration script.
+
+### Install required packages
+
+To get started, you'll need two different packages. The first is `algoliasearch`, the JavaScript client for the Algolia API, and the second is `meilisearch`, the JavaScript client for the Meilisearch API.
+
+```bash
+npm install algoliasearch meilisearch
+```
+
+### Create Algolia client
+
+You'll need your **Application ID** and **API Key** to start the Algolia client. Both can be found in your [Algolia account](https://www.algolia.com/api-keys).
+
+Paste the below code in `script.js`:
+
+```js
+const AlgoliaSearch = require("algoliasearch");
```

Comment:
In JS, capitalised names indicate functions that should be called with the `new` keyword. Though this is only a convention, I think we should not break it without a good reason.

```suggestion
const algoliaSearch = require("algoliasearch");
```